### ISSUE TITLE
Remove Duplicates from Sorted Array

### ISSUE THREAD

**Issue by AaryaZ** [gdscpce/hacktober-codesnippet#7]
Given an integer array `nums` sorted in non-decreasing order, remove the duplicates **in-place** such that each unique element appears only once. The relative order of the elements should be kept the same. Then return the number of unique elements in `nums`.

![image](https://github.com/gdscpce/hacktober-codesnippet/assets/70696659/e8311999-04db-41ac-9f01-2e3b1adf5635)

Follow and Read the Contributing Guidelines.
Only the assigned person is allowed to raise the PR on this Issue.
To get this issue assigned to you comment on this issue.

`Note:` The code can be written in any language of your choice.

**Comment by KESHAV4R3** [comment]
hii @AaryaZ  please assign this to me as it seems to be perfect  and compatable as per my domain

**Comment by Akash221104** [comment]
Name-Akash Satpute
brach-comps
Year-SE
Can u plz assign me this issue

**Comment by munirsiddiqui54** [comment]
hey @AaryaZ Please assign this issue to me.
I have the required Python solution for this issue.

Munir Siddiqui SY IT 

![image](https://github.com/gdscpce/hacktober-codesnippet/assets/122617589/1b05895f-8b11-4e1d-9acb-13abd71c406b)
![image](https://github.com/gdscpce/hacktober-codesnippet/assets/122617589/4fefb37c-c118-4294-b550-55133b28a944)

**Comment by AaryaZ** [comment]
@munirsiddiqui54 I appreciate your efforts. To keep things fair, the issue will be assigned to the person commenting first.
There are other issues as well to which you can contribute. You might as well create a new issue and raise the PR against it.

**Comment by munirsiddiqui54** [comment]
Okay it's fine aarya. Assign him this issue. I'll check other ones.

**Comment by KESHAV4R3** [comment]
> @munirsiddiqui54 I appreciate your efforts. To keep things fair, the issue will be assigned to the person commenting first. There are other issues as well to which you can contribute. You might as well create a new issue and raise the PR against it.
> 
> PS- Only those who comment their Name,Class,Year and College will be assigned the issue. This helps us differentiate students from other contributors.

But I was the first to ask this....

**Comment by KESHAV4R3** [comment]
Currently I am pursuing B-tech 3rd sem. from maharshi dayanand University in cse branch
